private_set_intersection/python/tests.py

```diff
@@ -50,6 +50,36 @@ def test_client_server(ds, reveal_intersection, duplicate):
         assert intersection <= (1.1 * len(client_items) / 2.0)
 
 
+@pytest.mark.parametrize("ds", [psi.DataStructure.GCS, psi.DataStructure.BLOOM_FILTER])
+@pytest.mark.parametrize("reveal_intersection", [False, True])
+@pytest.mark.parametrize("duplicate", [False, True])
+def test_large_client_small_server(ds, reveal_intersection, duplicate):
+    c = psi.client.CreateWithNewKey(reveal_intersection)
+    s = psi.server.CreateWithNewKey(reveal_intersection)
+
+    client_items = ["Element " + str(i) for i in range(1000)]
+    server_items = ["Element " + str(2 * i) for i in range(10)]
+
+    fpr = 1.0e-1
```

Comment:
Using a low `fpr` to 'catch' any failures in computing the intersection faster. It isn't that the values are important, but rather ensuring we catch any issues with increased cardinality.